packages/site-admin/src/router/components/link/stories/index.stories.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import { __experimentalVStack as VStack } from '@wordpress/components';
+/**
+ * Internal dependencies
+ */
+import { Link } from '../';
+import { browserHistory, RouterProvider, useLocation } from '../../../';
+import type { Meta, StoryObj } from '@storybook/react';
+import './style.stories.scss';
+
+const meta: Meta< typeof Link > = {
+	title: 'Components/Link',
+	component: Link,
+	tags: [ 'autodocs' ],
+	decorators: [
+		function WithRouterProvider( Story ) {
+			return (
+				<RouterProvider routes={ [] } pathArg="p">
```

Comment:
Why is `routes` an empty array? Can we change this example to be more similar to real-world scenario?

Reply:
It's true it feels a bit wired. Anyway, having an empty routes array can be a valid example for a story?<br>We could also make the routes prop optional and default it to an empty array, too. We need to provider to pass the context  to the story.

Reply:
I'm sure it is possible to leave an empty array, although I believe that we should try to put together the most comprehensive (and close to standard use cases) examples for the consumers of the package. And in that sense, I imagine that it would be better to have a list of routes and navigate between them?

Reply:
Putting it all together will make the story more complicated to understand. Also, although clearly, the routes are a fundamental piece of the implementation, they are not part of the Link component.

The benefit of this component, and what we should highlight, is the usage of its options (NavigationOptions). The second story covers it.

I would suggest that we improve other stories to show the whole implementation in detail. We already have an issue for this.